metrics/tripperware.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Mark Nevill. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package http_metrics
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/mwitkow/go-httpwares"
+)
+
+// Tripperware returns a new client-side ware that exports request metrics.
+func Tripperware(reporter Reporter) httpwares.Tripperware {
```

Comment:
This might be obvious but it is worth to mention in comment to add this tripperware as FIRST one wrap others with it. Same with middleware